Controller/PayumController.php

```diff
@@ -2,18 +2,33 @@
 namespace Payum\Bundle\PayumBundle\Controller;
 
 use Payum\Core\Payum;
+use Payum\Core\Registry\RegistryInterface;
 use Payum\Core\Security\GenericTokenFactoryInterface;
 use Payum\Core\Security\HttpRequestVerifierInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 
 abstract class PayumController extends AbstractController
 {
+    /**
+     * @var Payum
+     */
+    private $payum;
+
+    /**
+     * PayumController constructor.
+     * @param RegistryInterface $payum
+     */
+    public function __construct(RegistryInterface $payum)
```

Comment:
There needs to be `Payum\Core\Payum` typehint. `Payum\Core\Registry\RegistryInterface` does not contain all methods controllers call.

Reply:
Still waiting for this.

Reply:
Oops that was clumsy ;) ... done